### PR TITLE
Added LooseBadLLP jet cleaning WP

### PR DESCRIPTION
**Pull request by Darij-Starko**
Changed jet cleaning such that the WP LooseBadLLP is implemented with an infoswitch.

### CODE REVIEW

**Comment by jbossios** [comment]
BTW, there is no need to close and open a new PR (as long as you push the changes to the same branch).

**Comment by Darij-Starko** [comment]
just pushed the merged changes from r22/master to the darij/jet-cleaning branch. Please let me know if there is anything else left.

**Comment by jbossios** [comment]
@Darij-Starko Thanks, could you please also take care of my suggestions above?

**Comment by Darij-Starko** [comment]
I'm sorry I don't follow. I merged the changes from the r22/master branch into the darij/jet-cleaning branch as you suggested above and implemented the infoswitch changes as well.

**Comment by jbossios** [comment]
@Darij-Starko I'm sorry, I didn't realize I didn't submit my comments and they were still pending, you should be able to see them now.

**Comment by Darij-Starko** [comment]
Thank you @jbossios for sharing the comments. I believe I have addressed them all with the latest push to darij/jet-cleaning.

File: Root/JetContainer.cxx [hunk 1]
```diff
@@ -495,6 +498,9 @@ JetContainer::~JetContainer()
       delete m_clean_passLooseBad;
       delete m_clean_passTightBad;
     }
+    if(m_infoSwitch.m_cleanLLP) {
```
[the remark this answers is not in the document]
Comment (jbossios) [reply Root/JetContainer.cxx:501]: Thanks for changing this, I now realized that it was probably better to just move it outside, but I will resolve this and merge it anyway.

**Pull request merged**